app/templates/rooms.html

```diff
@@ -0,0 +1,7 @@
+<section ng-controller="RoomCtrl">
+    <p> {{ rooms.title }}</p>
+    <div ng-repeat="room in rooms track by $index">
```

Comment:
If you go with my suggestion above, `ng-repeat="room in rooms.allRooms track by $index"`